src/pages/Home.js

```diff
@@ -46,6 +47,15 @@ class Home extends React.Component {
         <ListItemText primary="Share what's new on your timeline"/>
       </ListItem>
       </div>
+      
+      {/* ECS: 
+      
+        This type of commented out code/feature that
+        wont be used or is part of an uninished feature
+        should not make it to the master branch. 
+      
```

Comment:
How do you leave out parts of the code in a script while still merging the parts you need to master?